Add doc for AKS integration

dejanu opened the pull request:
# What does this change
Add documentation concerning AKS integration.

# What issue does it fix
Closes #3173

schristoff commented:
Please don't forget to add your name to the `contributing.md` file! Thank you so much :)

dejanu commented:
> Please don't forget to add your name to the `contributing.md` file! Thank you so much :)
I added it the last time. Thx

kichristensen commented on docs/content/docs/integrations/aks.md:
````diff
@@ -4,4 +4,147 @@ description: How to connect to an AKS cluster inside a Porter bundle.
+* Bellow you can find a custom action called `merge` that it's used to autheticate in Azure and sets the desired subscription (that contains the resource group and AKS cluster) and uses [outputs](https://porter.sh/wiring/#outputs) to save the *kubeconfig* file:
+
+```bash
+merge:
````
I think it would make more sense to focus on connecting to AKS inside the bundle, for example by removing the custom action and putting it in the install action instead.

kichristensen commented on docs/content/docs/integrations/aks.md:
```diff
@@ -4,4 +4,147 @@ description: How to connect to an AKS cluster inside a Porter bundle.
+        - name: kubeconfig
+          path: newconf
+
+outputs:
```
I would suggest removing the output also to focus on the connection to AKS inside the bundle

dejanu replied:
removed outputs

kichristensen reviewed (changes requested):
Thank you so much for the PR, looks really good. I have added two small comments

dejanu commented:
@kichristensen, I've added Kubernetes mixin, just to illustrate the integration is working, hope it's ok. I've added a small video for demo purposes: https://youtu.be/lB1sjL55dZ0

kichristensen commented:
@dejanu Looks really good, thank you so much for the contribution
